src/services/ethereum/CustomSigner.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+
+import { Navigation } from 'react-native-navigation';
+import { screen } from '../../global/Screens';
+
+const ethers = require('ethers');
```

Comment:
Can you please use import instead of require? It looks more consistent with the whole app.

Reply:
Changed